inexor/fpsgame/weapon.cpp

```diff
@@ -711,7 +711,6 @@ namespace game
                 break;
 
             case GUN_SG:
-            {
```

Comment:
what's up with the formatting? Looks like we really need #417

Reply:
Removing these brackets doesn't work. https://travis-ci.org/inexorgame/inexor-core/jobs/270150983#L6596

Reply:
@Fohlen What is going on with #417 anyways, would you know?